packages/headless/src/controllers/facets/range-facet/date-facet/headless-date-facet.ts

```diff
@@ -55,15 +59,19 @@ export function buildDateFacet(
   return {
     ...rangeFacet,
     /**
-     * Selects (deselects) the passed value if unselected (selected).
-     * @param selection The facet value to select or deselect.
+     * Toggles the specified facet value.
+     * @param selection The facet value to toggle.
      */
     toggleSelect: (selection: DateFacetValue) =>
       dispatch(executeToggleDateFacetSelect({facetId, selection})),
 
-    /** @returns The state of the `DateFacet` controller.*/
```

Comment:
Why do we not need the `returns` here but we do at line 28 in the file above? (https://github.com/coveo/ui-kit/pull/376/files#diff-0aac8d5633bc178ffcc78e2b06d8fcadf8cd25c61d5aec7a2bfa150f7e2ff556R28)

Reply:
ah nevermind, I didn't realize that above it was for options 👍